Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 2019-11-19 -- v4.1.9
+- [changed] Moved reliable message queue delete operation to a serial queue to avoid race conditions in unit tests. (#4236)
```

Comment:
Should "reliable" be here?   It seems like it's mixing a feature description adjective with a description of a fix.

Reply:
Done.